politeiawww/api/v1/v1.go

```diff
@@ -300,3 +303,42 @@ type PolicyReply struct {
 	ValidMIMETypes   []string `json:"validmimetypes"`
 	ErrorCode        StatusT  `json:"errorcode,omitempty"`
 }
+
+// SubmitComment sends a comment from a user to a specific proposal.  Note that
+// the user is implied by the session.
+type SubmitComment struct {
```

Comment:
Can we use the same naming scheme as the other routes/commands? So for example, it would be `NewComment`, `RouteNewComment`, `handleNewComment()`, etc.

Reply:
Done